### ISSUE TITLE
the wordnet interface search is slightly broken

### ISSUE THREAD

**Issue by fcbond** [bond-lab/IMI#17]
def:word-in-definition is not working
html entities are escaped
the hidden features are not documented:
* you can use glob
* you can search in definitions
* you can search for synset IDs
@kopicee can you take a look at this?

**Comment by kopicee** [comment]
Regarding html entities: the issue is probably not due to wn-gridx.cgi
There is probably another script that generates a link like so
```
"/wn-gridx.cgi?lang=eng&ss=&lemma=%s&pos=x&gridmode=ntumcgrid" % html.escape("'")
```
And if cgi.FieldStorage.getfirst() does not interpret/unescape this then what should have been `'` is treated as a literal `&#x27;` (which seems to be the case).
Investigation is ongoing, I will write up a separate issue for this when I have a clearer idea what's up.

**Comment by kopicee** [comment]
Okay... looks like the var was just escaped twice in wn-gridx.cgi

**Comment by kopicee** [comment]
Double-escaped html issue fixed with c034bb0f2f1959685fe5680eaa17919be6de5d6b

**Comment by kopicee** [comment]
Noting that the bugs with searching `def::foo` on wn-gridx have been fixed and documentation added, this issue should be closable now
